argusclient/client.py

```diff
@@ -511,10 +511,10 @@ def delete(self, entity_id, permission):
 
 def convert(input):
     if isinstance(input, Mapping):
-        return {convert(key): convert(value) for key, value in input.iteritems()}
+        return {convert(key): convert(value) for key, value in input.items()}
     elif isinstance(input, list):
         return [convert(element) for element in input]
-    elif isinstance(input, basestring):
+    elif isinstance(input, str):
```

Comment:
This is not accurate in Python2, what is the reason behind this change?

Reply:
Hari, Used `String_types` from `six` package.